x-pack/plugins/cases/public/components/case_view/metrics/status.tsx

```diff
@@ -118,6 +118,7 @@ const CreationDate: React.FC<{ date: string }> = React.memo(({ date }) => {
     <FormattedRelativePreferenceDate
       data-test-subj={'case-metrics-lifespan-creation-date'}
       value={date}
+      stripMs={true}
     />
   );
 });
```

Comment:
On the same file on line 213 (in `ValueWithExplanationIcon` component) we use the `FormattedDate` component which renders a tooltip. So we have a tooltip that contains multiple tooltips. I think we should change to use `PreferenceFormattedDate` with `stripMs={true}`. I know it has nothing to do with this PR but I noticed it and I think is nice to fix it in this PR.

Reply:
@cnasikas can you open an issue for that and I'll open a new one? that way I can merge this one.

Reply:
Oops 😬